public/app/plugins/panel/status-history/StatusHistoryTooltip2.tsx

```diff
@@ -95,7 +95,6 @@ export const StatusHistoryTooltip2 = ({
         !field ||
         field === xField ||
         field.type === FieldType.time ||
-        field.type !== FieldType.number ||
```

Comment:
Thanks for fixing the issue I encountered while testing. Are there other types other than time we want to bypass?

Reply:
Looks like not, I was just chatting with @leeoniya about this and it should be ok the way it is.